app/main/main.js

```diff
@@ -130,6 +138,7 @@ app.on('ready', () => {
   });
   registerFileProtocol('libs', url => path.normalize(path.join(root, 'libs', url)));
 
+  storage.open('/tmp/foo.db');
```

Comment:
Probably should use path module here so it works in windows; also Electron has an [API](http://electron.atom.io/docs/latest/api/app/#appgetpathname) for getting temp directories in a cross platform way that we could use here